src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -51,6 +51,8 @@ static constexpr std::string_view LegacyToggleRetroEffectKey{ "toggleRetroEffect
 static constexpr std::string_view ToggleShaderEffectsKey{ "toggleShaderEffects" };
 static constexpr std::string_view MoveTabKey{ "moveTab" };
 static constexpr std::string_view BreakIntoDebuggerKey{ "breakIntoDebugger" };
+static constexpr std::string_view FindNextKey{ "findNext" };
+static constexpr std::string_view FindPrevKey{ "findPrev" };
```

Comment:
(Sorry if we had this discussion before)
Should we make next/prev an action arg? The design would be a bit weird, but maybe it could look like this:
```js
// this one already exists, but I put it here for reference
{ "command": "find", "keys": "ctrl+shift+f" },

// this is the proposed design
{ "command": { "action": "findMatch", "direction": "next" } },
{ "command": { "action": "findMatch", "direction": "previous" } },
```

For reference, here's what VS Code does:
![VS Code - Find actions](https://user-images.githubusercontent.com/11050425/106197262-56c1cf80-6167-11eb-82a0-c421bd3a1c5c.png)
It doesn't look like they have action args though.

Reply:
Yea sure. There's been no real design discussion, but that seems reasonable.